tutorials/postgres_object_1_intro.ipynb

```diff
@@ -385,8 +385,6 @@
     "## SQL Primer\n",
     "This optional section is intended for those with little or no prior experience with an SQL database, in particular with a server-mediated database like PostgreSQL. Most of the material discussed here applies to SQLite as well but not all; differences are pointed out. Since the PostgreSQL database is read-only for users write operations like `INSERT` and `UPDATE` are not discussed.\n",
     "\n",
-    "**NOTE:** The most up-to-date version of the Primer and following sections may be found [in Confluence](https://confluence.slac.stanford.edu/x/s4joE)\n",
```

Comment:
are you intended to remove this link to the Primer in this notebook?

Reply:
No, I must have committed the wrong version somehow.  Will fix.

Reply:
reverted to previous version.